db/schema.rb

```diff
@@ -10,14 +10,31 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[8.0].define(version: 2024_11_21_144711) do
-  create_sequence "qualifications_public_id_seq", start: 120000
+ActiveRecord::Schema[8.0].define(version: 2024_12_03_151209) do
+  create_sequence "qualifications_public_id_seq", start: 30690
```

Comment:
Does this line normally change? Just a bit concerned as it is actually lower than the original start number for the sequence

Reply:
😕 I'll put it back how it was